### PR TITLE
add option to output yaml

### PR DESCRIPTION
**Pull request by dustymabe**
This is useful when trying to write machine configs:
https://github.com/openshift/machine-config-operator/tree/e9033544f42191376529efdb7a67deb5427efd1d#applying-configuration-changes-to-the-cluster

### CODE REVIEW

**Comment by dustymabe** [comment]
rebased on top of latest master. Marking as "Ready for review"

File: filetranspile [hunk 1]
```diff
@@ -320,12 +323,15 @@ def main():
...
+    if args.format == 'json':
+        if args.pretty:
+            ignition_out = json.dumps(
+                merged_ignition, sort_keys=True,
+                indent=4, separators=(',', ': '))
+        else:
+            ignition_out = json.dumps(merged_ignition)
     else:
-        ignition_out = json.dumps(merged_ignition)
+        ignition_out = yaml.dump(merged_ignition)
```
Comment (ashcrow) [inline filetranspile]: I _think_ `safe_dump()` may be the better options here.

**Review by ashcrow — COMMENTED** [review]
One note ... tested locally and it does what it should.

**Comment by dustymabe** [comment]
New version pushed to address comments.

**Review by ashcrow — APPROVED** [review]
Tested locally and works as expected :+1:

**Pull request merged**